### DISCUSSION TITLE
Is it possible to make `APITable` handling double click too?

### DISCUSSION THREAD

**Discussion by flixyudh** [facebook/docusaurus#10516] [Release feedback]
I usually use double click to copy paste text from site content especially for API `property`, and when I tried to double click https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter I notice that the table would be focus on top when first click, and it's blocking me to block the `property`. 

example:

https://github.com/user-attachments/assets/9ed2d1ee-b725-40eb-9cc6-74e303d56ece

is it possible to make the table handle such a case?

I was thinking to wrap a function into `timeout` for `onClick` props and `clearTimeout` whenever [ondblclick](https://www.w3schools.com/jsrEF/event_ondblclick.asp) is triggered, but I don't know if it's a good way to do as I'm still a beginner.

**Comment by slorber** [comment]
This `APITable` component is not a public component, so if we change this, it will only affect the UX of our own website. 

I'd be happy to improve that if you want to submit a PR, but a `setTimeout` with an arbitrary number if not an ideal solution, I'd prefer to find something else. Maybe when clicking anchors we should update the URL without scrolling, even on a single click.

It's not a priority for me to fix this, so please submit a PR if you want to try. But I'd rather not merge a hacky PR with a setTimeout solution 😅 If we delayed the scroll to detect a double click that might feel awkward too.

**Reply by flixyudh** [reply]
I've submitted  a [Pull Request](https://github.com/facebook/docusaurus/pull/10534), could you take a look @slorber ? it's my first time PR into open source project, so please enlighten me if I doing something wrong 😄
